server/eclipse-project/src/main/java/us/freeandfair/corla/model/ComparisonAudit.java

```diff
@@ -564,6 +586,31 @@ public void signalSampleUnaudited(final int the_count) {
     }
   }
 
+
+  /**
+   * Signals that a sample has been unaudited, if the CVR was selected
+   * for this audit.
+   *
+   * @param count The count of samples that have been unaudited simultaneously
+   * (for duplicates).
+   * @parma cvrID The ID of the CVR to unaudit
+   */
+  public void signalSampleUnaudited(final int count, final Long cvrID) {
+    final boolean covered = isCovering(cvrID);
+    final boolean targeted = contestResult().getAuditReason().isTargeted();
+
+    if (targeted && !covered) {
+      LOGGER.debug("[sampleUnaudited: Targeted but not covered.]");
```

Comment:
same

Reply:
Oh, and the name is `sampleUnaudited` rather than `signalSampleUnaudited`, if we're being consistent in this method.